docs/reference/nodes/workflow.md

```diff
@@ -0,0 +1,51 @@
+# Workflow Nodes
+The following nodes are used to define the inputs and outputs of a workflow and nest it within another workflow.
+
+## Input and Output Nodes
+*Input* and *Output* nodes are used to define the required inputs and outputs of a workflow to be executed externally. Setting these nodes is essential to be able to nest workflows or executing the workflow in [batch mode](../../getting_started/basics/batch.md). The custom plugs from these nodes, which should be of type `through`, will act as data pathways, linking information set externally in the workflow to the current workflow and viceversa. 
+
+<figure style="width:80%;" markdown>
+    <img src="images/Input_outputNodes.png" alt="Input & Output Nodes">
+    <figcaption><b>Figure 1</b>: Input & Output Nodes UI.</figcaption>
+</figure>
+
+## Workflow Node
+The *Workflow* node executes an external workflow. This node allows to nest workflows embedding the execution logic of entire Shift workflows in one single node. The node updates its plugs dynamically depending on the sourced workflow file to enable access to the referenced workflow input and output data.
+
+<figure markdown>
+    <img src="images/workflow.png" alt="Workflow Node">
+    <figcaption><b>Figure 2</b>: Workflow Node UI.</figcaption>
+</figure>
+
+### Inputs
+
+- **File**: This plug of type [FileIn](../nodes#plugs) is used to reference the workflow file (*.sft* extension) that will be executed by the node. When the `file` plug content is set, the node will automatically update by creating the input and output plugs corresponding to the sourced workflow inputs and outputs.

```

Comment:
Please add a section to explain the double click functionality

Reply:
Done